src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -143,7 +150,55 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     if (onTabSelected) {
       onTabSelected(index);
     }
-    setCurrentIndex(index);
+    setCurrentActiveIndex(index);
+  };
+
+  const onKeyDown = (event: React.KeyboardEvent<HTMLUListElement>) => {
+    // alt + left/right is used by browser / assistive technologies
+    if (event.altKey) {
+      return;
+    }
+
+    const isTabDisabled = (index: number) => {
+      const tab = labels[index];
```

Comment:
`labels` could be passed as strings, no? This would not work then

Reply:
That's why I'm checking `isValidElement` in next line